druid/src/app.rs

```diff
@@ -138,6 +140,12 @@ impl<T: Data + 'static> WindowDesc<T> {
         self
     }
 
+    /// Set the window size at creation
+    pub fn window_size(mut self, width: f32, height: f32) -> Self {
```

Comment:
Should this take width and height as it is, and then call Size::new() inside, or just take in a Size?

Reply:
Its more flexible to take in a Size, but when using a library it's slightly annoying to have to dig up more types to do something simple.

Reply:
I'm sympathetic to that. One work around we have is that there's a `From<(f64, f64)>` for `Size`, so the actual argument here can be `size: impl Into<Size>`, which will let people just pass a tuple.

`kurbo` is a foundational crate for `druid`, and is used all over the place. If this gets too painful we can consider a prelude at some point, but I tend to prefer being explicit about imports.